.travis.yml

```diff
@@ -1,15 +1,19 @@
 # whitelist
 branches:
   only:
```

Comment:
The indenting in this this file is inconsistent. I suggest standardizing on 2-space indents.